vendor/github.com/docker/docker/contrib/nnp-test/nnp-test.c

```diff
@@ -1,10 +0,0 @@
-#include <stdio.h>
```

Comment:
Interesting that go modules discards c files here; wondering if that's gonna be an issue 🤔 (for these, it's probably fine, I think these are only used for integration tests in moby, so vndr may have just been overly cautious)

Reply:
Yes, I don't think these can be recognised by `go build` in any way...